src/components/tiles/map-tile/floorplan.tsx

```diff
@@ -0,0 +1,76 @@
+'use client';
+import clsx from 'clsx';
+import { useState } from 'react';
+import { MdLocationPin } from 'react-icons/md';
+
+import { Map } from './map';
+
+const map = [
+  'Regisztrációs pult',
+  'Bosch',
+  'Nokia',
+  'ICF Tech',
+  'Sicontact',
+  'Silicon Labs',
+  'Interactive Brokers',
+  'Paks II.',
+  'Barré',
+  'Nova',
+  'KUKA',
+  'OTP',
+  'Knorr-Bremse',
+  'SCHIS',
+  'Paripa',
+  'BME Suborbitals',
+  'BME Formula Racing Team',
+  'Schdesign',
+  'Kir-dev',
+  'SEM',
+  'HA5KFU',
+  'LEGO',
+];
+
+export function Floorplan() {
+  const [active, setActive] = useState(0);
+  return (
+    <section className='container'>
+      <h4 className='mb-4 text-center text-4xl font-black md:text-6xl lg:col-span-2 lg:text-8xl'>Térkép</h4>
```

Comment:
It's bigger than any other text on the website, yet it's a h4

I think just use an h2 like on the other tiles